jenkins/Jenkinsfile

```diff
@@ -8,48 +6,53 @@ def echoBanner(msg) {
 
 pipeline {
 	agent { node { label 'lisk-service' } }
+	when {
```

Comment:
`when` is only allowed inside of a `stage` directive https://www.jenkins.io/doc/book/pipeline/syntax/#when